3rd_party/licenses/gcc-runtime-INFO.csv

```diff
@@ -1,2 +1,2 @@
 name,version,revision,url,license,copyright,sourceURL
-GCC Runtime Library,10.3.0,f00b5710a30f22efc3171c393e56aeb335c3cd39,https://gcc.gnu.org,GPL-3.0 WITH GCC-exception-3.1,"Copyright (C) 2021 Free Software Foundation, Inc.",http://ftpmirror.gnu.org/gcc/gcc-10.3.0/gcc-10.3.0.tar.gz
```

Comment:
NIT: We should update the revision here to be `b71f1de6e9cf7181a288c0f39f9b1ef6580cf5c8`